File: nathan/Problema0354/problema0354.py
frase = input("Digite uma frase: ")
novaFrase = " "
for i in frase:
    if i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u':
        novaFrase += i.upper()
    else:
        novaFrase += i.lower()
print(novaFrase)

File: nathan/Problema0354/problema0354.java
package Problema0354;

import java.util.Scanner;

public class problema0354 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Digite uma frase: ");
        String frase = input.nextLine();
        char v[] = frase.toCharArray();
        for (int i = 0; i < frase.length(); i++) {
            if(v[i] == 'a' || v[i] == 'e' || v[i] == 'i' || v[i] == 'o' || v[i] == 'u'){            
                v[i] = Character.toUpperCase(v[i]);
            }else{
                v[i] = Character.toLowerCase(v[i]);
            }
        }
        for (int i = 0; i < v.length; i++) {
            System.out.print(v[i]);
        }
    }
}
